CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## UPCOMING
```

Comment:
are we waiting on anything else? otherwise i suggest to use `## 2.2.0 - 2020-07-02` here. i have to leave in a sec, so won't tag anymore today, but can do it tomorrow